.devcontainer/post-create.sh

```diff
@@ -32,18 +89,39 @@ if [[ -z "$DYNATRACE_OAUTH_CLIENT_SECRET" ]]; then
   exit 1
 fi
 
-kind create cluster --config .devcontainer/kind-cluster.yaml --wait 300s
-kubectl cluster-info --context kind-predictive-kubernetes-scaling-demo
+if [[ -z "$DEPLOY_MODE" ]]; then
+  DEPLOY_MODE="git"
+fi
+
+if [[  "$DEPLOY_MODE" == "git" ]]; then
+  echo "Creating Cluster"
+  kind create cluster --config .devcontainer/kind-cluster.yaml --wait 300s
+  kubectl cluster-info --context kind-predictive-kubernetes-scaling-demo
+else
+  if [[ -z "$GITHUB_TOKEN" ]]; then
+    echo "Error: GITHUB_TOKEN not set."
+    exit 1
+  fi
+  if [[ -z "$CLUSTERNAME" ]]; then
+    echo "Error: CLUSTERNAME not set."
+    exit 1
+  fi
+
+  export GITHUB_TOKEN=$GITHUB_TOKEN
+  export CODESPACE_NAME=$CLUSTERNAME
+fi
 
 dynatrace/init.sh
 apps/init.sh
 
-# Creation Ping
-curl -X POST https://grzxx1q7wd.execute-api.us-east-1.amazonaws.com/default/codespace-tracker \
-  -H "Content-Type: application/json" \
-  -d "{
-    \"tenant\": \"$DYNATRACE_ENVIRONMENT_ID\",
-    \"repo\": \"$GITHUB_REPOSITORY\",
-    \"demo\": \"obslab-predictive-kubernetes-scaling\",
-    \"codespace.name\": \"$CODESPACE_NAME\"
-  }"
+if [[   "$DEPLOY_MODE" == "git"  ]]; then
```

Comment:
Shouldn't the ping be made in both cases? Maybe with an additional argument if the kind cluster is used